configs/components/puppet.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/puppet.git", "ref": "7db03e5a28e9c2499da6e34d6c38849cce399b12"}
+{"url": "git://github.com/puppetlabs/puppet.git", "ref": "d21c5a749a87cf4e4308880e5daf331ccf8fd49e"}
```

Comment:
Ah, it looks like retargeting this updated the puppet SHA.

Reply:
Yeah, we will need to keep that update out of this PR

Reply:
@melissa Would you mind rebasing this against the `5.3.1_release` branch?

Reply:
done.